src/core/src/main/java/org/apache/jmeter/report/processor/graph/AbstractGraphConsumer.java

```diff
@@ -230,6 +233,30 @@ private void setMaxResult(MapResultData result, String name, Double value) {
                         value.doubleValue())));
     }
 
+ /**
+     * Creates the group info for elapsed time percentile depending on jmeter
+     * properties.
+     *
+     * @param propertyKey
+     *            the property key
+     * @param defaultValue
+     *            the default value
+     * @param seriesName Series name
+     * @param valueSelector Graph value selector
+     * @param seriesSelector Graph series selector
+     * @return the group info
+     */
+    protected final GroupInfo createPercentileGroupInfo(String propertyKey,
+                                                        int defaultValue, String seriesName,
```

Comment:
It seems default is always 95, should this default be in one place, i.e. inside this method instead?

Reply:
Could do, so basically rework the method from
``` protected final GroupInfo createPercentileGroupInfo(String propertyKey,
                                                        int defaultValue, String seriesName,
                                                        AbstractGraphValueSelector valueSelector,
                                                        AbstractSeriesSelector seriesSelector)```

to 
``` protected final GroupInfo createPercentileGroupInfo(String propertyKey,
                                                        String seriesName,
                                                        AbstractGraphValueSelector valueSelector,
                                                        AbstractSeriesSelector seriesSelector)```

And have the 95 in a const referenced directly inside the method.

That allows the calls to that method to be tightened up then

```createPercentileGroupInfo(PERCENTILE_PROPERTY, 95,seriesName,valueSelector,seriesSelector);```

goes to 

```createPercentileGroupInfo(PERCENTILE_PROPERTY,seriesName,valueSelector,seriesSelector);```